doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,191 @@
+---
+uid: Overview.Mvux.Advanced.Commands
+---
+
+# Commands
+
+## Commands recap
+
+In the [overview](xref:Overview.Mvux.Overview) you've seen a brief introduction to how a method in the Model gets generated as a command in the Bindable Proxy Model.
+
```

Comment:
Let's start this section with a description of what ICommand is, and why it's useful for data binding (ie to bind to Command property on a Button).
Then we can talk about the AsyncCommand implementation that exists in MVUX